File: latihan3/php/processor.php
<?php
    class processor{
        // atribut
        private $price = 0;
        private $name = "blank";
     
        // constructor
        function __construct($price=0, $name="blank"){
            $this->price = $price;
            $this->name = $name;
        }

        // Setter dan Getter

        function setPrice($price){
            $this->price = $price;
        }
        function getPrice(){
            return $this->price;
        }

        function setName($name){
            $this->name = $name;
        }
        function getName(){
            return $this->name;
        }

        // destructor
        function __destruct(){}

    }
?>

File: latihan3/java/processor.java
public class processor {
    // atribut
    private int price;
    private String name;

    // constructor
    processor(){}

    processor(int price, String name){
        this.price = price;
        this.name = name;
    }

    // setter dan getter

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
    
    
}
